packages/metadata-crawler/src/crawler.ts

```diff
@@ -41,60 +49,60 @@ async function addVerificationClaimToDatabase(address: string, domain: string, v
 
     await client
       .query(query, values)
-      .catch((error) => console.error(`Database error ${error}\n query: ${query}`))
-      .then(() => console.log(`Verification flag added to domain ${domain} and address ${address}`))
+      .catch((error) => logger.error(`Database error ${error}\n query: ${query}`))
```

Comment:
Since we are using a structured logger, I would recommend passing parameters structured as well

Reply:
:+1: